staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/volumeoptions.go

```diff
@@ -90,21 +91,21 @@ func CheckControllerSupported(ctrlType string) bool {
 }
 
 // VerifyVolumeOptions checks if volumeOptions.SCIControllerType is valid controller type
-func (volumeOptions VolumeOptions) VerifyVolumeOptions() bool {
+func (volumeOptions VolumeOptions) VerifyVolumeOptions() error {
```

Comment:
@jsafrane  look gr8 👍🏽  

Just out of curiosity, this function only `validates` and does not `mutate`, why its not a `pointer` receiver ?

Reply:
I don't know :-)